src/daemon/dlt-log-levels.conf

```diff
@@ -0,0 +1,13 @@
+# Configuration for default log levels.
+# APPIDs in this configuration file overwrite the global ContextLogLevel level from dlt.conf
+# It can be used to prevent single applications from logging (for example due to logging sensitive data)
+# or to disable file transfers from an application etc.
+# Furthermore single applications can use a higher log level than the default.
+# If the context id is empty, the configuration will be valid for all app ids
```

Comment:
I think this is easy to misunderstand. I would phrase it like:
If the context id is empty, the configuratio nwill be valid for all context id's of the given application id.

Reply:
makes sense :+1: